docs/data/base/components/select/UseSelect.tsx

```diff
@@ -38,7 +38,6 @@ const Toggle = styled('div')(
       : `0 5px 13px -3px ${grey[200]}`
   };
   border-radius: 0.75em;
-  margin: 0.5em;
```

Comment:
IMHO, this should be banned. In almost all cases, the margin should be the owner of the parent component that does the layout.